build/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "short_name": "react-tooltip",
+  "name": "react-tooltip",
+  "start_url": "./index.html",
+  "display": "standalone",
+  "theme_color": "#000000",
+  "background_color": "#ffffff"
+}
```

Comment:
build directory was deleted from master, it is not necessary now.

Reply:
Removed